playbooks/roles/ocp-sso/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+# defaults file for playbooks/roles/ocp-sso
+sso_catalogsource: "redhat-operators"
+sso_directory: "/tmp/sso"
+sso_catalogsource_image: ""
+sso_upgrade_channel: "stable"
+sso_schedulerImage: "registry.k8s.io/scheduler-plugins/kube-scheduler:v0.30.6"
+sso_go_tarball: "https://go.dev/dl/go1.22.4.linux-ppc64le.tar.gz"
+sso_e2e: false
+sso_e2e_github_repo: ""
+sso_e2e_github_branch: "master"
+sso_github_username: ""
+sso_github_token: ""
+sso_cleanup: true
```

Comment:
These variables "sso_github_username", "sso_github_token", "sso_github_repo" can be kept empty if not running the E2E tests for this operator . Also "sso_catalogsource_image" is only required when testing pre-release version of this operator or else it can be kept empty.

Reply:
please mark these variables as required if they don't have any default value